esp8266-fastled-webserver/commands.cpp

```diff
@@ -16,6 +16,12 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "common.h"
```

Comment:
SIMPLE FILE REVIEW

File existed as commands.h in many of the active branches.

* Wrapped file with `#if defined(ENABLE_IR)` so becomes substantially empty when IR is not enabled.
* Moved function `handleIrInput()` from main `.ino` into this file
* Removed unused function `heldButtonHasBeenHandled()`
* Removed unused function `waitForIRCode()`